app/controllers/api/v1/invoice_items/invoice_items_inv_controller.rb

```diff
@@ -0,0 +1,8 @@
+class Api::V1::InvoiceItems::InvoiceItemsInvController < ApplicationController
```

Comment:
Can we fully name Inv to Invoice? I can see the point of abbreviating, but it's only four extra characters.